tools/create_operator.py

```diff
@@ -274,64 +274,48 @@ def __call__(
     # Operator implementation test
     with open(
         os.path.join(
-            "temporian",
-            "implementation",
-            "numpy",
-            "operators",
-            "test",
-            lower_op + "_test.py",
+            "temporian", "core", "operators", "test", f"test_{lower_op}.py"
         ),
         "w",
         encoding="utf-8",
     ) as file:
         file.write(
-            f"""\
-{license_content()}
+            f"""{license_content()}
 
-from absl.testing import absltest
+from absl.testing import absltest, parameterized
 
-import numpy as np
-from temporian.core.operators.{lower_op} import {capitalized_op}
 from temporian.implementation.numpy.data.io import event_set
-from temporian.implementation.numpy.operators.{lower_op} import (
-    {capitalized_op}NumpyImplementation,
-)
-from temporian.implementation.numpy.operators.test.utils import (
-    assertEqualEventSet,
-    testOperatorAndImp,
-)
+from temporian.test.utils import assertOperatorResult, f64, i32
+
 
-class {capitalized_op}OperatorTest(absltest.TestCase):
-    def setUp(self):
-        pass
+class {capitalized_op}Test(parameterized.TestCase):
+    def test_empty(self):
+        evset = event_set(timestamps=[], features={{"a": i32([])}})
+        result = evset.{lower_op}()
+        expected = event_set(timestamps=[], same_sampling_as=evset)
 
-    def test_base(self):
+        assertOperatorResult(self, result, expected, check_sampling=True)
+
+    @parameterized.parameters(
+        {{"in_timestamps": [0.0, 1.0], "out_timestamps": [0.0, 1.0]}},
+        {{"in_timestmaps": [1.0, 2.0], "out_timestamps": [1.0, 2.0]}},
+    )
```

Comment:
I think the most probable scenario is not using parameterized rather than using it? I'd leave it out of the template (and maybe add a commented line with the import or a sentence nudging the developer towards using it if needed?)

Reply:
I think it's a positive thing to induce ourselves to use parametrized tests, since you tend to add more test cases and edge cases. Also, the effort to remove this is negligible. Don't you agree?